targets/wasm_exec.js

```diff
@@ -298,9 +298,22 @@
 
 					// func finalizeRef(v ref)
 					"syscall/js.finalizeRef": (v_ref) => {
-						// Note: TinyGo does not support finalizers so this should never be
-						// called.
-						console.error('syscall/js.finalizeRef not implemented');
+						// Note: TinyGo does not support finalizers so this is only called
+						// for one specific case, by js.go:jsString. and can/might leak memory.
+						const id = mem().getUint32(unboxValue(v_ref), true);
```

Comment:
```suggestion
						const id = v_ref & 0xffffffffn;
```

Based on @prochac suggestion.

Reply:
@ldemailly I can confirm this works (tested with: https://github.com/rsc/qr/tree/master/qart)